doc/examples.py

```diff
@@ -98,3 +103,46 @@ def print_hand_result(hand_result):
 
 result = calculator.estimate_hand_value(tiles, win_tile, config=config)
 print_hand_result(result)
+
+####################################################################
+# Bug: Yakuman and Non-yakuman Yakus should not add together       #
```

Comment:
This can be moved to `doc/test.py`. In example file we can keep only examples how to work with lib.